src/collectors/systemd-journal.plugin/README.md

```diff
@@ -1,472 +1,629 @@
-journal centralization servers.
+The `systemd` journal plugin provides an efficient way to view, explore, and analyze `systemd` journal logs directly from the Netdata dashboard. It combines powerful filtering, real-time updates, and visual analysis tools to help you troubleshoot system issues effectively.
 
-![image](https://github.com/netdata/netdata/assets/2662304/691b7470-ec56-430c-8b81-0c9e49012679)
+![Netdata systemd journal plugin interface](https://github.com/netdata/netdata/assets/2662304/691b7470-ec56-430c-8b81-0c9e49012679)
 
-## Key features
+---
 
-- Works on both **individual servers** and **journal centralization servers**.
-- Supports `persistent` and `volatile` journals.
-- Supports `system`, `user`, `namespaces` and `remote` journals.
-- Allows filtering on **any journal field** or **field value**, for any time-frame.
-- Allows **full text search** (`grep`) on all journal fields, for any time-frame.
-- Provides a **histogram** for log entries over time, with a breakdown per field-value, for any field and any
-  time-frame.
-- Works directly on journal files, without any other third-party components.
-- Supports coloring log entries, the same way `journalctl` does.
-- In PLAY mode provides the same experience as `journalctl -f`, showing new log entries immediately after they are
-  received.
+## Key features
 
-### Prerequisites
+- **Unified view of logs** from multiple sources (system, user, namespace, remote)
+- **Real-time streaming** with PLAY mode for continuous monitoring
+- **Powerful filtering** on all journal fields with counters showing matching entries
+- **Full-text search** with wildcard and pattern matching
+- **Visual analysis** with interactive histograms showing log frequency
+- **Enriched field display** for improved readability (priorities, UIDs, timestamps)
+- **High performance** with intelligent sampling for large datasets
+- **Zero configuration** for immediate use on supported systems
+- **Multi-node support** for centralized log analysis
+- **UI-based exploration** without needing to learn complex journalctl syntax
+- **Integrated with Netdata's dashboard** for correlation with metrics
 
-`systemd-journal.plugin` is a Netdata Function Plugin.
+---
 
-To protect your privacy, as with all Netdata Functions, a free Netdata Cloud user account is required to access it.
-For more information, check [this discussion](https://github.com/netdata/netdata/discussions/16136).
+## Prerequisites
 
-### Limitations
+| Requirement | Details |
+|-------------|---------|
+| Netdata Agent v1.44+ | This plugin requires Netdata version 1.44 or newer |
+| `libsystemd` | The plugin relies on libsystemd to read journal files |
+| Netdata Cloud account | Required to use Netdata Functions including this plugin |
+| **Not supported in**: | Alpine Linux Docker containers or static builds (use Debian-based containers instead) |
```

Comment:
```suggestion
| Netdata Agent v1.44+ | This plugin requires Netdata version 1.44 or newer |
| Netdata Cloud account | Required to use Netdata Functions, including this plugin |
| **Not supported in**: | Static builds (use Debian-based containers instead) |
```

Reply:
- The package `libsystemd-dev` (for Debian-based systems) or `systemd-devel` (for RPM-based systems) is required only when building Netdata from source. Since this isn't an officially supported installation method, we can exclude it from our docs.
- We don't provide Docker containers based on Alpine Linux.